### PR TITLE
Fix null error in modal & adapters 

### PR DESCRIPTION
**Pull request by JosefForkman**
Maybe we need to merge in #10  before we do this PR

### CODE REVIEW

File: src/components/EventModal/components/DropdownSelect.tsx [hunk 1]
```diff
@@ -22,34 +24,38 @@ const DropdownIcon = ({
...
-                    {dropDownValue || (
-                        <Input.Placeholder>{selectedOption}</Input.Placeholder>
-                    )}
-                </InputBase>
-            </Combobox.Target>
-            <Combobox.Dropdown>
-                {children}
-            </Combobox.Dropdown>
-        </Combobox>
+        <>
```
Comment (sebastianandreasson) [inline src/components/EventModal/components/DropdownSelect.tsx]: was there a reason to wrap this in a empty fragment? otherwise it should just be reverted

**Review by sebastianandreasson — CHANGES_REQUESTED** [review]
Looks good, just needed to run prettier on the changed files because they were not formatted. Just one question regarding wrapping a component, I was not sure if this was needed or not so did not change it myself. With the change its ok to merge

File: src/components/EventModal/components/DropdownSelect.tsx (hunk 1 above)
Comment (JosefForkman) [reply src/components/EventModal/components/DropdownSelect.tsx]: I experimented with adding a label but then saw that mantine supports it. I forgot to remove it. Now it should be gone.

Comment (JosefForkman) [reply src/components/EventModal/components/DropdownSelect.tsx]: I experimented with adding a label but then saw that mantine supports it. I forgot to remove it. Now it should be gone.

**Pull request merged**